src/index.html

```diff
@@ -11,7 +11,182 @@
+
+        <label class="form-field">
+          How old are You?
+          <input
+          type="number"
+          name="age"
+          required
+          min="1"
+          max="100"
+          value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+          type="date"
+          name="dob"
+          required
+          >
+        </label>
+
+        <div>
+          <label class="form-label"
+            for="terms-and-conditions">
+            I accept the term of the agreement
+          </label>
+          <input
+            id="terms-and-conditions"
+            type="checkbox"
+            name="terms"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          required
+          placeholder="email@example.com"
+          autocomplete="off"
+          >
+        </label>
+
+        <label>
+          Password:
+          <input
+          type="password"
+          name="password"
+          required
+          autocomplete="off"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>
+          An interesting fact about you!
+        </legend>
+
+        <label class="form-field">
+          Do you love cats?
+          <label>
+            <input
+            type="radio"
+            name="cats"
+            value="y"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+            type="radio"
+            name="cats"
+            value="n"
+            >
+            No
+          </label>
+        </label>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+          type="color"
+          name="color"
+          required
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+          type="time"
+          name="time"
+          required
+          >
+        </label>
+
+        <label class="form-field">
+        What are your favorite brand of cars?
+          <select
+          name="color"
+          required
+          multiple
+          size="4">
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label>
+          How do you rate our work?
+          <input
+          type="range"
+          name="rating"
+          value="0"
+          required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Additional info:</legend>
+        <label class="form-field">
+          Comments:
+          <textarea
+          name="review"
+          autocomplete="off"
+          >
+          </textarea>
```

Comment:
use this trick to avoid empty space as I show in screenshot when click on textarea
```suggestion
          ></textarea>
```
<img width="359" alt="Screenshot 2023-01-02 at 12 24 29 PM" src="https://user-images.githubusercontent.com/60295799/210219041-4454a040-2264-4bd7-a955-05fa096cf06e.png">

Reply:
After this change, index.html does not pass the 'npm run lint test'. 
![image](https://user-images.githubusercontent.com/121205546/210258914-474209f6-2b63-43f8-8ec7-918b35803ea8.png)
![image](https://user-images.githubusercontent.com/121205546/210258966-a26aeb87-ccc8-4be5-bf26-aea681c1b634.png)

Reply:
Try to use like this
`
<textarea
  name="review"
  autocomplete="off"
 ></textarea>
 `